### PR TITLE
update readme file

### PR DESCRIPTION
**Pull request by NitishKSaini**
## Review
my update readme file

### CODE REVIEW

**Comment by NitishKSaini** [comment]
Let me Know the response.

**Comment by nitish-saini** [comment]
> ## Review
> my update readme file

ok 👍

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
-# Demo-Website
\ No newline at end of file
+# Demo-Website
+1. This is my demo website
```
Comment (NitishKSaini) [inline README.md:2]: he there

File: README.md [hunk 2]
```diff
@@ -1 +1,3 @@
-# Demo-Website
\ No newline at end of file
+# Demo-Website
```
Comment (NitishKSaini) [inline README.md:1]: hello

**Comment by nitish-saini** [comment]
## OK
👍

**Pull request merged**